static/js/util.js

```diff
@@ -246,6 +246,15 @@ export function convert_message_topic(message) {
     }
 }
 
+function is_url_previewable(href) {
+    const github_re =
+        /(?:(?:https?:\/\/)?(?:www.)?)?github.com\/([^/]*)\/([^/]*)\/(issues|pull)\/([^#/?^|]*)/;
+    if (github_re.test(href)) {
+        return true;
+    }
```

Comment:
Let's replace this regular expression with actually parsing the URL using the URL library. See https://developer.mozilla.org/en-US/docs/Web/API/URL.